website/content/docs/deprecation/faq.mdx

```diff
@@ -46,6 +47,31 @@ This workaround will fail in a future version of Go, however, the Go team has no
 
```

Comment:
This will be quite a tricky thing for users to verify they are ready to upgrade. Might it be helpful if we give them some tips for checking existing certificates before the upgrade, and/or e.g. some log lines they should expect to see if they hit this deprecation?